src/Phing/RdfPurge.php

```diff
@@ -49,7 +49,7 @@ public function setSolrUrl($solr_url) {
    * @param string[] $solr_cores
    *   A comma delimited list of Solr core names.
    */
-  public function setSolrCores($solr_cores) {
+  public function setSolrCores(array $solr_cores) {
```

Comment:
Actually this is not correct. The `$solr_core` var is a string (then the docs is wrong s/string[]/string/). That is the value we are passing in the .xml file.